eslintrc.base.js

```diff
@@ -71,7 +71,7 @@ module.exports = {
         'react/prop-types': 'off',
         'react/display-name': 'off',
         'react/no-unescaped-entities': 'off',
-        'react/jsx-key': 'off',
+        'react/jsx-key': 'error',
```

Comment:
Having entries in this code block that aren't disabling rules feels inconsistent with the comment at the top of the block. I would recommend instead deleting this line (since `react/jsx-key` is already an error by default in the `plugin:react/recommended` set we extend) and moving the other two `error` cases below it up to the "additional rules not enabled by default" section

Reply:
Thanks for pointing those headings out, I skipped over them. Fixed in [560b41d](https://github.com/microsoft/accessibility-insights-web/pull/5792/commits/560b41d17fb8cd8e72e98bad3613078bfe998989)!